src/_style.scss

```diff
@@ -296,8 +296,15 @@ select.form-control {
 .tooltip-shadow {
   box-shadow: 0px 5px 15px 0px rgba(0,0,0,0.3) !important;
 }
+
+.password-tooltip {
+  font-size: 12px;
+  line-height: 2;
+}
+
```

Comment:
Can't we use the `.x-small` class directly on the tooltip? 
https://paragon-openedx.netlify.app/foundations/typography